.github/workflows/static-analysis-psalm.yml

```diff
@@ -0,0 +1,102 @@
+name: PHP Static Analysis with Psalm
```

Comment:
We might want to split these two static analysis actions into another action (or maybe two) that acts as a composite action where we do the setup.

Reply:
Performance wise this is probably a good idea. But I'm not that good with github actions

Reply:
We can leave it like this for now, we just have to make sure that if we change something in the duplicate code we must change it in both actions.